skycoin/skywire
# improve on survey and log collecting

Did you run `make format && make check`? Yes

Fixes #_

 Changes:	
- increase timeout of IP request
- improve logic of generate survey
- build-ui
- improve logic of log collecting
- add two flag to log collecting
  - `ut` to get custom UT address, usable for get specific visor(s) data, like `https://ut.skywire.skycoin.com/uptimes?v=v2&visors=0353e0fe09ab53b9e66da7ca25d338db5aaf3d530a2bb1e22daebf2caf4ba89cf2,0353f35ad1c960a398b5e0c0c9c7a091c8536eff1977a1f72380c753b885fdb3a7`
  - `batchSize` for set count of visor in each batch, default value is 50 there

How to test survey part of this PR:
- run a visor
- set reward address, both available in UI and CLI, then check survey

How to test log collecing part of this PR:
- run `skywire-cli log collecting --all` or/and set `--ut` flag for specific visor
